### PR TITLE
soapyrtlsdr: 0.3.0 -> 0.3.3

### PR DESCRIPTION
**Pull request by luizribeiro**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by winterqt — CHANGES_REQUESTED** [review]
Please change your commit message according to the [manual](https://nixos.org/manual/nixpkgs/unstable/#submitting-changes-making-patches), i.e. `soapyrtlsdr: 0.3.0 -> 0.3.3`.

**Comment by winterqt** [comment]
Do you mind if I push a small commit cleaning something up to this branch?

**Comment by luizribeiro** [comment]
@winterqt, nope! Go for it :)

**Review by winterqt — APPROVED** [review]

**Comment by winterqt** [comment]
@luizribeiro Since the maintainer of this package seems to be MIA based on their GitHub activity, would you like to pick up the mantle?

**Comment by luizribeiro** [comment]
> @luizribeiro Since the maintainer of this package seems to be MIA based on their GitHub activity, would you like to pick up the mantle?

Done! Also followed your advice and fixed darwin on this PR itself. Let me know if I messed something up :)

File: pkgs/applications/radio/soapyrtlsdr/default.nix [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
+    ++ lib.optionals stdenv.isDarwin [ libobjc IOKit Security ];
 
   cmakeFlags = [ "-DSoapySDR_DIR=${soapysdr}/share/cmake/SoapySDR/" ];
 
   meta = with lib; {
     homepage = "https://github.com/pothosware/SoapyRTLSDR";
     description = "SoapySDR plugin for RTL-SDR devices";
     license = licenses.mit;
-    maintainers = with maintainers; [ ragge ];
+    maintainers = with maintainers; [ luizribeiro ragge ];
```
Comment (winterqt) [inline pkgs/applications/radio/soapyrtlsdr/default.nix:34]: From what I've seen, it's more common to add new maintainers after the existing ones.

```suggestion
    maintainers = with maintainers; [ ragge luizribeiro ];
```

**Review by winterqt — CHANGES_REQUESTED** [review]
A few (minor) things:

1. `soapyrtlsdr: Fix build on Darwin` -> `soapyrtlsdr: fix build on darwin` (or Darwin, it doesn't really matter, what matters is the lowercase `f`)
2. Split the maintainers commit into two, one adding yourself as a maintainer (`maintainers: add luizribeiro`), and one adding yourself as a maintainer to the package (i.e. `soapyrtlsdr: add luizribeiro  as maintainer`)

File: pkgs/applications/radio/soapyrtlsdr/default.nix (hunk 1 above)
Comment (luizribeiro) [reply pkgs/applications/radio/soapyrtlsdr/default.nix:34]: makes sense, I went with alphabetical order :P

**Review by winterqt — APPROVED** [review]

**Pull request merged**